app/scripts/proactive/templates/workflow-variables-view-template.html

```diff
@@ -191,7 +191,7 @@ <h4 class="modal-title"><%- jobModel.Name %> </h4>
                         model = data;
                         },
                         error: function(xhr, ajaxOptions, thrownError) {
-                            alert("Could not fetch model from url '" + urlToFetch + "': " + xhr.responseText);
+                            console.error("Could not fetch model from url '" + urlToFetch + "': " + xhr.responseText);
```

Comment:
is there a console.warn ?

Reply:
@fviale there is an error that will be displayed anyway by the browser in the console for an 404 or 403 network call

Reply:
ok